tests/test_study.py

```diff
@@ -559,3 +559,58 @@ def test_storage_property():
 
     study = optuna.create_study()
     assert study.storage == study._storage
+
+
+@pytest.mark.parametrize('n_jobs', [1, 4])
+def test_callbacks(n_jobs):
+    # type: (int) -> None
+
+    study = optuna.create_study()
+
+    def objective(trial):
+        # type: (optuna.trial.Trial) -> float
+
+        return trial.suggest_int('x', 1, 1)
+
+    # Empty callback list.
+    study.optimize(objective, callbacks=[], n_trials=10, n_jobs=n_jobs)
+
+    # A callback.
+    values = []
+    callbacks = [lambda study, trial: values.append(trial.value)]
+    study.optimize(objective, callbacks=callbacks, n_trials=10, n_jobs=n_jobs)
+    assert values == [1 for _ in range(10)]
```

Comment:
I think this assertion can be simplified like `assert values == [1] * 10`. We can see this style in [study.py](https://github.com/pfnet/optuna/blob/master/optuna/study.py#L456), so please think of rephrasing this line.

Reply:
Cool!
Fixed: https://github.com/pfnet/optuna/pull/480/commits/c3e0961a1181d5313fa34415adbc0feb99c30f2b